torch/functional.py

```diff
@@ -369,8 +369,15 @@ def meshgrid(*tensors, indexing: Optional[str] = None) -> Tuple[Tensor, ...]:
             tensors (list of Tensor): list of scalars or 1 dimensional tensors. Scalars will be
                 treated as tensors of size :math:`(1,)` automatically
 
-            indexing: (str, optional): the indexing mode requested.
-                Only "ij" is currently supported.
+            indexing: (str, optional): the indexing mode, either "xy" or "ij".
```

Comment:
Specify which is the default

Reply:
The warning above needs to be updated because the 'indexing' mode is now supported

Reply:
Is `torch.cartesian_prod` analogous to indexing mode ij or xy?

Reply:
> The warning above needs to be updated because the 'indexing' mode is now supported

pushed that update to the previous PR. In this PR, I drop the note that "xy" will be supported.

Reply:
> Specify which is the default

done. i also point to the warning to remind about upcoming changes.

Reply:
> Is `torch.cartesian_prod` analogous to indexing mode ij or xy?

The operations are analogous. But it is more trivially transformable to "ij" mode. Do you think a note about this is required? *I do not*. I think the comparison is more conceptual and beneficial because cartesian product is a very well known. But utility-wise, the operations are not meant to be interchangeable.

Reply:
If you think both modes are analogous it's fine